demo/addons/fmod/FMODDebugPanel.gd

```diff
@@ -0,0 +1,17 @@
+extends Panel
+
+onready var dsp_usage = get_node("%DSPUsage")
+onready var mem_usage = get_node("%MemUsage")
+onready var rms_val = get_node("%RMSValue")
+
+func _physics_process(delta):
```

Comment:
Why in physics_process and not regular process ?
Also, don't tie the process running or not with visibility. As it is, you will deactivate the process by hiding the panel but it won't reactivate when you make it visible again